external-crates/move/crates/move-compiler/tests/sui_mode/linter/true_negative_share_owned.move

```diff
@@ -0,0 +1,106 @@
+// object is re shared, but it is never transferred and doesn't have public transfer
```

Comment:
Is there an .exp file missing for this one?

Reply:
No, they are negative cases so they do not trigger the lint